src/lexer.coffee

```diff
@@ -482,7 +529,12 @@ exports.Lexer = class Lexer
   # Use a `\` at a line-ending to suppress the newline.
   # The slash is removed here once its job is done.
   suppressNewlines: ->
-    @tokens.pop() if @value() is '\\'
+    prev = @prev()
+    if prev[1] is '\\'
+      if prev.comments
+        # By definition, there must be a token before a `\` token.
```

Comment:
Damn, talk about edge case. There’s no sensible reason to start a file with a line continuation, is there? Or even to start a line with a line continuation? I’m thinking about just discarding the comments in such a case, as it seems foolish to handle.